tools/run_tests/performance/scenario_config.py

```diff
@@ -413,8 +413,8 @@ def scenarios(self):
         unconstrained_client='sync')
 
     yield _ping_pong_scenario(
-        'python_protobuf_async_streaming_qps_unconstrained', rpc_type='STREAMING',
-        client_type='ASYNC_CLIENT', server_type='SYNC_SERVER',
+        'python_protobuf_sync_streaming_qps_unconstrained', rpc_type='STREAMING',
+        client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
         unconstrained_client='async')
```

Comment:
Yes, I've tried it locally.  The streaming client doesn't spin up a thread per outstanding request, it just queues up the number of outstanding requests before it starts reading.